pkg/config/piped.go

```diff
@@ -971,6 +972,14 @@ func (n *NotificationReceiverSlack) Validate() error {
 		formatMentionedAccount := strings.TrimPrefix(mentionedAccount, "@")
 		mentionedAccounts = append(mentionedAccounts, formatMentionedAccount)
 	}
+	mentionedGroups := make([]string, 0, len(n.MentionedGroups))
+	for _, mentionedGroup := range n.MentionedGroups {
+		formatMentionedGroup := fmt.Sprintf("<!subteam^%s>", strings.TrimPrefix(mentionedGroup, "@"))
+		mentionedGroups = append(mentionedGroups, formatMentionedGroup)
+	}
```

Comment:
Is this logic necessary here?  It seems also in the func `getGroupsAsString` 👀
 If possible, it would be nice to put it in the `getGroupsAsString` because this func is just for validating. :)

Reply:
correct me if im wrong, they are difference thing `mentionedGroups` is for [NotificationReceiverSlack](https://pipecd.dev/docs-v0.47.x/user-guide/managing-piped/configuration-reference/#notificationreceiverslack) under piped config

the remain for deployment mention slack group (a new field) 
https://pipecd.dev/docs-v0.47.x/user-guide/configuration-reference/#notificationmention

Reply:
um not sure but i've seen we already have same thing for user
```
	mentionedAccounts := make([]string, 0, len(n.MentionedAccounts))
	for _, mentionedAccount := range n.MentionedAccounts {
		formatMentionedAccount := strings.TrimPrefix(mentionedAccount, "@")
		mentionedAccounts = append(mentionedAccounts, formatMentionedAccount)
	}
```
should we already remove it as well because `getAccountsAsString` already have it?